components/CardList/CardList.js

```diff
@@ -1,8 +1,18 @@
 import CardContainer from "../CardContainer/CardContainer";
 
-export default function CardList({ initialData }) {
+export default function CardList({ initialData, cards }) {
   return (
     <>
+      {cards
```

Comment:
nice solution with the ternary operator!

You could access if it is better to join the array or the inititalData and the array of cards in the _app.js already (inside of a function) and only pass one dataSet to the CardList component. 
But for now, this works absolutely fine!

Reply:
(check my comment in the _app.js for details)